pkg/harvester/edit/harvesterhci.io.host/index.vue

```diff
@@ -398,16 +406,50 @@ export default {
     },
 
     async saveLonghornNode() {
+      const inStore = this.$store.getters['currentProduct'].inStore;
+
       const disks = this.longhornNode?.spec?.disks || {};
 
       this.newDisks.map((disk) => {
         (disks[disk.name] || {}).tags = disk.tags;
       });
 
-      try {
-        await this.longhornNode.save();
-      } catch (err) {
-        return Promise.reject(exceptionToErrorsArray(err));
+      let count = 0;
+
+      const retrySave = async() => {
+        try {
+          await this.longhornNode.save();
+        } catch (err) {
+          if ((err.status === 409 || err.status === 403) && count < 3) {
+            count++;
+
+            await this.$store.dispatch(`${ inStore }/find`, {
+              type: LONGHORN.NODES,
+              id:   this.longhornNode.id,
+              opt:  { force: true },
+            });
+
+            await new Promise(resolve => setTimeout(resolve, '5000'));
+            await retrySave();
+          } else {
+            return Promise.reject(exceptionToErrorsArray(err));
+          }
+        }
+      };
+
+      await retrySave();
+    },
+
+    async actuallySave(url) {
+      if ( this.isCreate ) {
+        url = url || this.schema.linkFor('collection');
+        const res = await this.value.save({ url });
+
+        if (res) {
+          Object.assign(this.value, res);
+        }
+      } else if (!isEqual(this.originValue, this.value)) {
+        await this.value.save();
```

Comment:
Is this function being used? I think it's the same as the method in the mixin.

Reply:
Done